spec/sidekiq/decision_review/submit_upload_spec.rb

```diff
@@ -230,5 +230,217 @@
+          drea = DecisionReviewEvidenceAttachment.new(
+            guid: appeal_submission_upload.decision_review_evidence_attachment_guid
+          )
+          drea.set_file_data!(file)
+          drea.save!
+          drea
+        end
+      end
+
+      before do
+        appeal_submission.update(type_of_appeal: 'SC')
+      end
+
+      it 'calls the documents service api with file body and document data (for SC)' do
+        VCR.use_cassette('decision_review/SC-GET-UPLOAD-URL-200_V1') do
+          VCR.use_cassette('decision_review/SC-PUT-UPLOAD-200_V1') do
+            VCR.use_cassette('decision_review/200_pdf_validation') do
+              allow(Rails.logger).to receive(:info)
+              expect(Rails.logger).to receive(:info).with({
+                                                            message: 'Evidence upload retrieval success!',
+                                                            user_uuid: appeal_submission.user_uuid,
+                                                            action: 'Evidence upload retrieval',
+                                                            form_id: '995',
+                                                            upstream_system: 'AWS S3',
+                                                            downstream_system: nil,
+                                                            is_success: true,
+                                                            http: {
+                                                              status_code: nil,
+                                                              body: nil
+                                                            },
+                                                            appeal_submission_upload_id: appeal_submission_upload.id,
+                                                            form_attachment_id: attachment.id
+                                                          })
+              expect(Rails.logger).to receive(:info).with({
+                                                            message: 'Get lighthouse evidence upload url success!',
+                                                            user_uuid: appeal_submission.user_uuid,
+                                                            action: 'Get lighthouse evidence upload url',
+                                                            form_id: '995',
+                                                            upstream_system: 'Lighthouse',
+                                                            downstream_system: 'Lighthouse',
+                                                            is_success: true,
+                                                            http: {
+                                                              status_code: 200,
+                                                              body: anything # why isn't this working?
```

Comment:
for some reason this `anything` matcher isn't working even though it's pretty much the same test as the one above for NOD — anyone have ideas for what might be happening here?

Reply:
nvm i figured out what was actually not matching up!